src/jqLite.js

```diff
@@ -671,7 +671,10 @@ forEach({
       }
       return element.value;
     }
-    element.value = value;
+    // Workaround for Firefox validation #12102.
+    if (element.value != value) {
```

Comment:
It solves it as long as the initial value of `ngModel` is not the same as the `input[value]` (in most cases we expect that `input[value]` to be an empty string, so yes, just doing `!==` should be fine (famous last words)